src/elements/List/ListItem.js

```diff
@@ -50,11 +104,48 @@ ListItem.propTypes = {
   /** Additional classes. */
   className: PropTypes.string,
 
+  /**
+   * Shorthand for primary content.
+   *
+   * Heads up!
+   *
+   * This is handled slightly differently than the typical `content` prop since
+   * the wrapping ListContent is not used when there's no icon or image.
+   *
+   * If you pass content as:
+   * - an element/literal, it's treated as the sibling node to
+   * header/description (whether wrapped in Item.Content or not).
+   * - a props object, it forces the presence of Item.Content and passes those
+   * props to it. If you pass a content prop within that props object, it
+   * will be treated as the sibling node to header/description.
+   */
```

Comment:
I pushed an update to the `ComponentProps` component to preserve line breaks.  That way this doesn't render as a single ran together line in the prop table.